tests/tracking/fluent/test_fluent_autolog.py

```diff
@@ -172,23 +167,21 @@ def test_universal_autolog_calls_pyspark_immediately():
 
 
 @pytest.mark.parametrize("config", [{"disable": False}, {"disable": True}])
-def test_universal_autolog_attaches_pyspark_import_hook_if_pyspark_isnt_installed(config):
+def test_universal_autolog_attaches_pyspark_import_hook_in_oss(config):
     with mock.patch("mlflow.spark.autolog", wraps=mlflow.spark.autolog) as autolog_mock:
         autolog_mock.integration_name = "spark"
-        # simulate pyspark not being installed
-        autolog_mock.side_effect = ImportError("no module named pyspark blahblah")
 
         mlflow.autolog(**config)
-        autolog_mock.assert_called_once()  # it was called once and failed
+        autolog_mock.assert_not_called()
 
         # now the user installs pyspark
         autolog_mock.side_effect = None
 
         mlflow.utils.import_hooks.notify_module_loaded(pyspark)
 
-        # assert autolog is called again once pyspark is imported
-        assert autolog_mock.call_count == 2
-        assert autolog_mock.call_args_list[1] == config
+        # assert autolog is called once pyspark is imported
+        assert autolog_mock.call_count == 1
+        assert autolog_mock.call_args_list[0] == config
```

Comment:
Now we can use `assert_called_once_with` here.

Reply:
If I use `assert autolog_mock.assert_called_once_with(**config)`, then it raise error 
```
E           AssertionError: expected call not found.
E           Expected: autolog(disable=False)
E           Actual: autolog()
```
seemingly it cannot handle default arguments.